src/Dapper.AOT.Analyzers/SqlAnalysis/TSqlProcessor.cs

```diff
@@ -175,6 +175,9 @@ protected virtual void OnTableVariableUsedAsScalar(Variable variable)
     protected virtual void OnNullLiteralComparison(Location location)
         => OnError($"Null literals should not be used in binary comparisons; prefer 'is null' and 'is not null'", location);
 
+    protected virtual void OnPseudoPositionalParameter(Location location)
+        => OnError($"Dapper.Vanilla does not handle pseudo-positional parameters correctly", location);
```

Comment:
A: unnecessary $ ?
B: it is more like Dapper will incorrectly treat this literal *as* a pseudo-positional parameter

Reply:
Removed `$` from other `OnError()` usages as well and changed the description

Reply:
and changed that in `DapperAnalyzer.Diagnostic` as well